### PR TITLE
337 org contact fields

### PR DESCRIPTION
**Pull request by DutchBen**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoForms/formFields/ContactPersonNameField.tsx [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+              )
+              .filter(
+                  (item) =>
+                      !chosenPersons.some(
+                          (person) => person.email === item.email,
+                      ),
+              )
+        : [];
+
+    useEffect(() => {
```
Comment (Georgi2704) [inline packages/orchestrator-ui-components/src/components/WfoForms/formFields/ContactPersonNameField.tsx:127]: Maybe you can use react-query here ?

File: packages/orchestrator-ui-components/src/components/WfoForms/formFields/ContactPersonNameField.tsx [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+        if (organisationIdValue) {
+            apiClient
+                .axiosFetch<ContactPerson[]>(
+                    `/surf/crm/contacts/${organisationIdValue}`,
+                    {},
+                    {},
+                    false,
+                )
+                .then((result) => {
+                    console.log('fetched contacts');
```
Comment (tjeerddie) [inline packages/orchestrator-ui-components/src/components/WfoForms/formFields/ContactPersonNameField.tsx:137]: remove console log

**Review by tjeerddie — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoForms/formFields/ContactPersonNameField.tsx (hunk 2 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WfoForms/formFields/ContactPersonNameField.tsx:137]: fixed

File: packages/orchestrator-ui-components/src/components/WfoForms/formFields/ContactPersonNameField.tsx (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WfoForms/formFields/ContactPersonNameField.tsx:127]: There are a few ways we get data now and that's a good point. Let's make that one of the points to discuss Thursday. I'm going to leave it like this now

**Pull request merged**